src/platform/FailSafeContext.cpp

```diff
@@ -35,16 +35,17 @@ void FailSafeContext::HandleArmFailSafe(System::Layer * layer, void * aAppState)
 
 void FailSafeContext::CommissioningFailedTimerComplete()
 {
-    // TODO: If the fail-safe timer expires before the CommissioningComplete command is
-    // successfully invoked, conduct clean-up steps.
-
     ChipDeviceEvent event;
-    event.Type                         = DeviceEventType::kCommissioningComplete;
-    event.CommissioningComplete.Status = CHIP_ERROR_TIMEOUT;
-    CHIP_ERROR status                  = PlatformMgr().PostEvent(&event);
+    event.Type                                                 = DeviceEventType::kCommissioningComplete;
+    event.CommissioningComplete.PeerFabricIndex                = mFabricIndex;
+    event.CommissioningComplete.AddNocCommandHasBeenInvoked    = mAddNocCommandHasBeenInvoked;
+    event.CommissioningComplete.UpdateNocCommandHasBeenInvoked = mUpdateNocCommandHasBeenInvoked;
```

Comment:
Don't we need to initialize these members in the other place where kCommissioningComplete events are created?

Reply:
We do not use these two fields in other cases, I updated other places to keep consistency. .